src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -282,222 +248,151 @@ function WorkspaceMembersPage(props) {
-                                }}
-                                onSelectRow={() => toggleUser(item.accountID, item.pendingAction)}
-                            />
-                        </View>
-                        {(props.session.accountID === item.accountID || item.role === 'admin') && (
-                            <View style={[styles.badge, styles.peopleBadge]}>
-                                <Text style={[styles.peopleBadgeText]}>{props.translate('common.admin')}</Text>
-                            </View>
-                        )}
-                    </PressableWithFeedback>
-                    {!_.isEmpty(errors[item.accountID]) && (
-                        <FormHelpMessage
-                            isError
-                            message={errors[item.accountID]}
-                        />
-                    )}
-                </OfflineWithFeedback>
-            );
-        },
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-        [selectedEmployees, errors, props.session.accountID, dismissError, toggleUser],
-    );
-
     const policyOwner = lodashGet(props.policy, 'owner');
     const currentUserLogin = lodashGet(props.currentUserPersonalDetails, 'login');
-    const removableMembers = {};
-    let data = [];
-    _.each(props.policyMembers, (policyMember, accountID) => {
-        if (isDeletedPolicyMember(policyMember)) {
-            return;
-        }
-        const details = props.personalDetails[accountID];
-        if (!details) {
-            Log.hmmm(`[WorkspaceMembersPage] no personal details found for policy member with accountID: ${accountID}`);
-            return;
-        }
-        data.push({
-            ...policyMember,
-            ...details,
-        });
-    });
-    data = _.sortBy(data, (value) => value.displayName.toLowerCase());
-    data = getMemberOptions(data, searchValue.trim().toLowerCase());
-
-    // If this policy is owned by Expensify then show all support (expensify.com or team.expensify.com) emails
-    // We don't want to show guides as policy members unless the user is a guide. Some customers get confused when they
-    // see random people added to their policy, but guides having access to the policies help set them up.
-    if (policyOwner && currentUserLogin && !PolicyUtils.isExpensifyTeam(policyOwner) && !PolicyUtils.isExpensifyTeam(currentUserLogin)) {
-        data = _.reject(data, (member) => PolicyUtils.isExpensifyTeam(member.login || member.displayName));
-    }
-
-    _.each(data, (member) => {
-        if (member.accountID === props.session.accountID || member.login === props.policy.owner || member.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) {
-            return;
-        }
-        removableMembers[member.accountID] = member;
-    });
     const policyID = lodashGet(props.route, 'params.policyID');
     const policyName = lodashGet(props.policy, 'name');
 
+    const getListData = () => {
```

Comment:
This method still returns member options right we can keep the same name `getMemberOptions`

Reply:
For sure - done ✅